README.md

```diff
@@ -32,6 +34,73 @@ composer require yiisoft/rbac-rules-container --prefer-dist
 
 ## General usage
 
+### Direct interaction with rules container
+
+```php
+use Psr\Container\ContainerInterface;
+use Yiisoft\Rbac\Item;
+use Yiisoft\Rbac\RuleInterface;
+use Yiisoft\Rbac\Rules\Container\RulesContainer;
+
+/**
+ * Checks if user ID matches `authorID` passed via parameters.
+ */
+final class AuthorRule implements RuleInterface
+{
+    public function execute(string $userId, Item $item, array $parameters = []): bool
+    {
+        return $parameters['authorID'] === $userId;
+    }
+}
+
+$rulesContainer = new RulesContainer(new MyContainer());
+$rule = $rulesContainer->create(AuthorRule::class);
+```
+
+- `MyContainer` is a container for resolving dependencies and  must be an instance of 
+`Psr\Container\ContainerInterface`.  [Yii Dependency Injection](https://github.com/yiisoft/di) implementation also can 
```

Comment:
```suggestion
- `MyContainer` is a container for resolving dependencies and must be an instance of 
`Psr\Container\ContainerInterface`. [Yii Dependency Injection](https://github.com/yiisoft/di) implementation also can 
```

Reply:
Fixed in https://github.com/yiisoft/rbac-rules-container/pull/6#discussion_r845453019.